src/modules/ekf2/EKF2.cpp

```diff
@@ -131,6 +128,13 @@ EKF2::EKF2(bool multi_mode, const px4::wq_config_t &config, bool replay_mode):
 	_param_ekf2_rng_a_igate(_params->range_aid_innov_gate),
 	_param_ekf2_rng_qlty_t(_params->range_valid_quality_s),
 	_param_ekf2_rng_k_gate(_params->range_kin_consistency_gate),
+	_param_ekf2_rng_pos_x(_params->rng_pos_body(0)),
+	_param_ekf2_rng_pos_y(_params->rng_pos_body(1)),
+	_param_ekf2_rng_pos_z(_params->rng_pos_body(2)),
+	_param_ekf2_terr_mask(_params->terrain_fusion_mode),
+	_param_ekf2_terr_noise(_params->terrain_p_noise),
+	_param_ekf2_terr_grad(_params->terrain_gradient),
```

Comment:
What if we still want the terrain estimate using optical flow? I think those parameters should only be removed if rng and flow are disabled

Reply:
I was going to do terrain estimator separately, but at the moment EKF2_OPTICAL_FLOW depends on EKF2_RANGE_FINDER and I was thinking we should just extract the terrain estimator into a separate class anyway.